composer.json

```diff
@@ -64,24 +64,21 @@
         }
     },
     "scripts": {
-        "analyse": [
-            "@composer validate --strict",
-            "vendor/bin/phpstan analyse -c phpstan.neon -l max",
-            "vendor/bin/ecs check src/ spec/"
```

Comment:
Why are you removing the ECS?
However, if we are removing the running of ECS, we should add this to a separate build, triggered even once a week, wdyt? 🤔